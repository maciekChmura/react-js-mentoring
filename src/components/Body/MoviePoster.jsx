import React from 'react';
import styled from 'styled-components';

const PosterWrapper = styled.div`
  width: 200px;
  height: 300px;
  overflow: hidden;
`;

const Img = styled.img`
  width: 100%;
`;

const MoviePoster = props => {
  return (
    <PosterWrapper>
      <Img src={props.posterImage} />
    </PosterWrapper>
  );
};

export default MoviePoster;
