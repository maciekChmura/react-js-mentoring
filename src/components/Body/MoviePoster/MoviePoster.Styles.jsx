import styled from 'styled-components';

export const PosterWrapper = styled.div`
  width: 200px;
  height: 300px;
  overflow: hidden;
`;

export const Img = styled.img`
  width: 100%;
`;
