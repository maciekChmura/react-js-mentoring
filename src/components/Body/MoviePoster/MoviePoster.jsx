import React from 'react';
import { PosterWrapper, Img } from './MoviePoster.Styles';

const MoviePoster = ({ posterImage }) => (
  <PosterWrapper>
    <Img src={posterImage} />
  </PosterWrapper>
);

export default MoviePoster;
