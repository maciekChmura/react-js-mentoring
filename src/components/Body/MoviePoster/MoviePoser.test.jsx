import React from 'react';
import renderer from 'react-test-renderer';
import MoviePoster from './MoviePoster';

test('MoviePoster renders', () => {
  const component = renderer.create(<MoviePoster />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
