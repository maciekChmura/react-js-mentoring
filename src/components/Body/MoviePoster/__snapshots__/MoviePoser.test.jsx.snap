// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MoviePoster renders 1`] = `
<div
  className="MoviePosterStyles__PosterWrapper-sc-2d9g7n-0 fjhBTm"
>
  <img
    className="MoviePosterStyles__Img-sc-2d9g7n-1 fjLIup"
  />
</div>
`;
